src/main/java/no/sikt/nva/scrapers/EmbargoScraper.java

```diff
@@ -46,7 +48,11 @@ public static Map<String, List<Embargo>> getEmbargoes(File file) {
     }
 
     private static boolean fileIsEmpty(File file) {
-        return file.length() == 0;
+        return file.length() == 0 || fileHasNotContent(file);
+    }
+
+    private static Boolean fileHasNotContent(File file) {
+        return attempt(() -> Files.readString(Path.of(file.toURI())).isBlank()).orElse(failure -> false);
```

Comment:
StringUtils.isBlank instead of string.isBlank, then we don't get any NullpointerExceptions